rbac/management/group/view.py

```diff
@@ -225,6 +225,17 @@ def protect_system_groups(self, action, group=None):
             error = {key: [_(message)]}
             raise serializers.ValidationError(error)
 
+    def restrict_custom_default_group_renaming(self, request, group):
+        """Restrict users from changing the name or description of the Custom default group."""
+        method = request.method
+        invalid_methods = {"PUT"}
+        is_updating = method in invalid_methods
+        if group.platform_default and not group.system and is_updating:
+            key = "Custom default group"
```

Comment:
is to possible to keep this consistent across app ? key is `detail` - it refers to this https://datatracker.ietf.org/doc/html/rfc7807#section-3.1